libs/platform/experience/src/services/layout/plugins/types/collapsible/collapsible.model.ts

```diff
@@ -6,10 +6,13 @@ declare global {
   export interface Layouts {
     collapsible: undefined;
   }
-  // eslint-disable-next-line @typescript-eslint/no-empty-interface
-  export interface LayoutProperty extends CollapsibleLayoutProperties {}
-}
 
-export interface CollapsibleLayoutProperties {
```

Comment:
Did we release this interface already, dunno BC or not?

Reply:
no this was added earlier this week